src/test/unit/motor_output_unittest.cc

```diff
@@ -28,6 +28,41 @@ extern "C" {
 #include "unittest_macros.h"
 #include "gtest/gtest.h"
 
+extern "C" {
+	uint8_t atomic_BASEPRI;
+
+	bool featureIsEnabled(uint8_t f);
+	float getDigitalIdleOffset(void *cfg);
+	float scaleRangef(float a, float b, float c, float d, float e);
+
+}
```

Comment:
```suggestion
```

Seems to me that the "extern" macro got closed prematurely

Reply:
fixed